src/components/InvitationModal/stylesheet.scss

```diff
@@ -0,0 +1,245 @@
+.invitation-modal {
+    margin: -20px;
+}
+
+.invitation-modal-content {
+    text-align: center;
+    color: white;
+}
+
+.email-block {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    padding-bottom: 5px;
+}
+
+// form {
+//     display: flex;
+//     flex-direction: column;
+// }
+
+//Style for email input
+
+.email-input {
+    display: flex;
+    flex-direction: column;
+}
+
+input {
+    width: 282px;
+    height: 24px;
+}
+
+input[type=email] {
+    padding: 15px;
+    border-radius: 6px;
+    background-color: rgb(35, 35, 35);
+}
+
+input[type=email]:focus {
+    border: 1px solid rgba(255, 255, 255, 0.5);
+    outline: none
+}
+
+.valid-email {
+    text-align: start;
+    color: rgba(34, 181, 49, 0.75);
+    font-size: 12px;
+    font-weight: bold;
+    padding-top: 3px;
+    opacity: 1;
+}
+
+.invalid-email {
+    text-align: start;
+    color: rgba(255, 0, 0, 0.5);
+    font-size: 12px;
+    font-weight: bold;
+    padding-top: 3px;
+    opacity: 1;
+}
+
+.email:has(+.invalid-email) {
+    border: 1px solid rgba(255, 0, 0, 0.5);
+
+}
+
+input:invalid {
+    border: 1px solid rgba(255, 0, 0, 0.5);
+}
+
+.dropdown {
+    width: 282px;
+}
+
+#recent-invites {
+    appearance: none;
+    width: 282px;
+    height: 24px;
+    min-width: 282px;
+    background-color: rgb(51, 51, 51);
+}
+
+.options {
+    width: 282px;
+}
+
+.send-button {
+    width: 86px;
+    height: 24px;
+    font-size: 14px;
+    color: white;
+    border-radius: 6px;
+    background-color: #C56E5B;
+    border: none;
+    margin-top: 4px;
+}
+
+button:hover {
+    background-color: #E2944B;
+}
+
+p {
+    padding: 20px;
+    padding-bottom: 0px;
+}
```

Comment:
I think there are too many hard-code values like this. Could you come up with a way to reduce this (for example, creating a container div around `p`, `email-block`, etc. to make sure everything is padded/margined/sized properly)?